flink/README.md

```diff
@@ -236,6 +236,8 @@ TiBigDate/Flink does not support the following features:
 | tidb.cluster-jks-trust-path                 | _                                                                              | The path of the JKS trust certificates store, which is used to authenticate remote service. e.g. `/home/TiBigData/.ci/config/cert/jks/server-cert-store`.                                                                                                                                                                                                                                                                                                 |
 | tidb.cluster-jks-trust-password             | _                                                                              | The key of JKS trust certificates store.                                                                                                                                                                                                                                                                                                                                                                                                                  |
 | tidb.telemetry.enable                       | true                                                                           | Whether to enable telemetry collection in TiBigData. Telemetry can be off by setting `false`.                                                                                                                                                                                                                                                                                                                                                             |
+| tidb.cached-rows-in-memory                  | false                                                                          | Whether to cache data in memory for each region. This helps improve the success rate when reading data, but takes up more memory(100-200m).                                                                                                                                                                                                                                                                                                               |
```

Comment:
Is it associated with https://docs.pingcap.com/tidb/dev/tune-region-performance#use-region-split-size-to-adjust-region-size. For big regions, it's quite dangerous.

Reply:
Should we check and limit the size? If the size > 200M, we will skip this feature.